big-practice/src/js/dashboard.js

```diff
@@ -36,11 +91,56 @@ const renderTable = () => {
 
       tableBody.innerHTML = tableTemplate;
     })
+    // Display error message when retrieving data from Json server
     .catch((error) => alert('An error occurred while getting movie', error));
 };
 
+/**
+ * Create new movie and save to database
+ */
+const createMovie = (data) => {
+  const option = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  };
+
+  fetch(moviesApi, option)
+    // Parses JSON response into native JavaScript objects
+    .then((response) => response.json())
+    .then(() => renderTable())
+    // Show error message when API call is wrong
+    .catch((error) => alert('An error occurred while creating movie', error));
+};
+
+const handleCreateForm = () => {
+  const name = nationInput.value;
+  const director = directorInput.value;
+  const nation = nationInput.value;
+  const formData = { name, director, nation };
+
+  createMovie(formData);
+};
+
+const createForm = () => {
+  if (isValidForm()) {
+    handleCreateForm();
+    renderTable();
```

Comment:
You render Table 2 times, in createForm and in createMovie. Pls remove one of them